kconfig-distro-sof-dev-update.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+KCONFIG_DIR=$(dirname ${BASH_SOURCE[0]})
+echo $KCONFIG_DIR
+
+. $KCONFIG_DIR/kconfig-lib.sh
+
+echo sof > localversion
```

Comment:
I think it should be `echo -sof > localversion` (adds `-`), that looks a little better and fits in with convention (from what I can tell).

Reply:
yep, will fix.